django/utils/version.py

```diff
@@ -2,8 +2,18 @@
 import functools
 import os
 import subprocess
+import sys
 from distutils.version import LooseVersion
 
+# Private, stable API for detecting the Python version. PYXY means "Python X.Y
+# or later". So that third-party apps can use these values, each constant
+# should remain as long as the oldest supported Django version supports that
+# Python version.
```

Comment:
If the API is private, why worry about third-party compat? Doesn't private mean "only used by Django core"?

Reply:
My thinking is that "private" means "undocumented." Since there isn't much maintenance burden with this policy, I think it's okay.

Reply:
Sure, just wanted to be clear